scripts/tasks.py

```diff
@@ -6,7 +6,7 @@
 import numpy as np
 import itertools
 
-logging.basicConfig(level=logging.DEBUG)
+logging.basicConfig(level=logging.INFO)
```

Comment:
Unrelated to the PR, but maybe this should be an option we can change from the command line?

Reply:
Yes I considered that but got lazy - please make an issue or a PR of it.